swagger/docs/css/print.css

```diff
@@ -0,0 +1,1172 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
You can't add Swagger-UI as a dependency, only Swagger. Swagger requires that you use their middleware on all of your routes.